scheduler/indexer/awesome/do_http_get.go

```diff
@@ -0,0 +1,23 @@
+package awesome
```

Comment:
this should be in a common package or something. Do other indexers require different http get methods?